.circleci/config.yml

```diff
@@ -26,8 +26,7 @@ jobs:
           name: lint
           command: |
             go get github.com/golangci/golangci-lint/cmd/golangci-lint
-            echo ${CIRCLE_BRANCH}
-            golangci-lint run --new-from-rev=${CIRCLE_BRANCH} ./...
+            golangci-lint run --new-from-rev=origin/master
```

Comment:
Here I've added possible to ignore most existing problems. 
I expect `golangci-ling` will be running only with diff

Reply:
Also, I'll have provided a commit with fixing the most critical issues found by linters

Reply:
Yep, we can create an issue to fix the rest and label it a `good new starter issue` or such.